spec/services/draw_activator_spec.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+require 'spec_helper'
```

Comment:
hmm. I think that any tests that persist data will be Rails-specific, so it's probably fine how it is?